modules/ROOT/pages/am-installing.adoc

```diff
@@ -136,7 +136,7 @@ Enabling Monitoring can impact CPU utilization and memory utilization, which var
 
 == What to do Next
 
-- If you find connection failure or retry entries, ensure that ports 443 and 5044 in your outbound firewall are open.
+- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open. Older runtimes might still use port 5044 and that needs to be open as well, if that's the case.
```

Comment:
```suggestion
- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open. If an older runtime uses port 5044, the port must remain open.
```

Reply:
Are connection failure/retry entries log entries? Should specify throughout the topic.

Reply:
Not really. If the runtime is using the old port for some reason, we should mention checking both ports (443/5044) to be open

Reply:
@sarathecoubian Any updates here?

Reply:
@juanpablotimpanaro Click "commit suggestion".

Reply:
If that change is okay with you, @juanpablotimpanaro , please commit it. This change has been requested by some users over Slack, so we should make sure it gets merged soon.